gcm_filters/filter.py

```diff
@@ -83,6 +83,7 @@ def _compute_filter_spec(
             else:  # ndim==2
                 n_steps = np.ceil(6.4 * filter_scale / dx_min).astype(int)
 
+    print("n_steps = %i" % n_steps)
```

Comment:
No print statements in the code.

Reply:
Thanks for catching this! The print statement is now removed.